doc/example/example_ordinal.ipynb

```diff
@@ -0,0 +1,1047 @@
+      "text/plain": [
+       "<Figure size 1850x1050 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from pypesto.visualize import waterfall\n",
+    "\n",
+    "waterfall([res_standard, res_reduced, res_reduced_reparameterized])"
```

Comment:
Assuming you use the same starts for each result: adding `waterfall(..., order_by_id=True)` changes the waterfall plot to ensure the same starts are plotted at the same `x-positions`. Won't change your result here since there appears to be no difference between settings. The waterfall plot says they're all the same but it's difficult to tell. Perhaps a table of fvals makes this clearer?

Reply:
Didn't know about this setting, neat. Will use it :D 
I added a listing of fvals as well for exactness.